### PR TITLE
Added show changelog information icon

### PR DESCRIPTION
**Pull request by mjobst-necls**
## Description
A information icon was added to the admin updater section. This icon is only visible if a new version is available. The webpage that is opened in a new window by clicking the icon is related to an url with the following scheme:

- https://owncloud.org/changelog/#latest8.2   - if new version is 8.2.x
- https://owncloud.org/changelog/#latest9.1   - if new version is 9.1.x
- and so on

If there is no such anchor available, the changelog page opens anyway.

## Related Issue
#26796

## Motivation and Context
This change improves the user (admin) experience

## How Has This Been Tested?
local installation on current master branch

## Screenshots (if appropriate):
screenshot from 2016-12-07 15-59-48

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- No such test available

### CODE REVIEW

**Comment by mjobst-necls** [comment]
@VicDeo @PVince81 please check.

**Comment by DeepDiver1975** [comment]
Nice! I like it!

**Comment by PVince81** [comment]
@VicDeo please review

File: apps/updatenotification/templates/admin.php [hunk 1]
```diff
@@ -19,6 +19,11 @@
 	<?php if($isNewVersionAvailable === true): ?>
 		<strong><?php p($l->t('A new version is available: %s', [$newVersionString])); ?></strong>
 		<input type="button" id="oca_updatenotification_button" value="<?php p($l->t('Open updater')) ?>">
+		<a target="_blank" rel="noreferrer" class="icon-info svg" title="<?php p($l->t('Show changelog')); ?>" href="https://owncloud.org/changelog/#latest
+		<?php 
+			$versionParts = explode('.', explode(' ', $newVersionString)[1]); // remove the 'ownCloud' prefix
```
Comment (mjobst-necls) [inline apps/updatenotification/templates/admin.php]: @VicDeo done

**Comment by VicDeo** [comment]
@DeepDiver1975
Jenkins ist tot: `mkdir(): No space left on device`

**Comment by PVince81** [comment]
@VicDeo tests passed now. Merge ?

**Comment by davitol** [comment]
:+1: Tested in several browsers and WFM

**Pull request merged**